sections/resources.tex

```diff
@@ -34,7 +34,7 @@
 \centering\footnotesize\textcolor{darkcandyapplered}{\textbf{\textit{Resource Production Tracker}}}
 \bigskip
 
-Possible Resource Die\index{Resource Die} \includesvg[height=10px]{\svgs/resource_die.svg} results:
+Possible \textbf{Resource Die}\index{Resource Die}\index{Dice} \includesvg[height=10px]{\svgs/resource_die.svg} results:
```

Comment:
I like the index entry but I don't think `textbf` is necessary.

Reply:
In theory I agree. But if I'm coming here via the link from Index if I want to see the die, it's easier to spot the die if it's bold. But having an image of the die would be better than textbf. I just don't have the image - would need to find one somewhere...

Reply:
Fair. Let me take a look.